assignments/arrays.js

```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === 33 ){
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}` );
+    }
+}
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar = inventory.pop();
+console.log(lastCar);
```

Comment:
so at first glance this looked cool and intuitive, however, Array.prototype.pop() also removes the last element from the array, so while you were able to receive the data of the last element in the array, you also removed it from the array.  I don't think that this was you're initial intention with this however it's good to be conscious of this. There is another way to get the last element that I'll let you figure out unless you need help of course.